internal/server/instance/drivers/driver_qemu.go

```diff
@@ -6278,13 +6278,13 @@ func (d *qemu) Export(w io.Writer, properties map[string]string, expiration time
 	defer revert.Fail()
 
 	// Check for Direct I/O support.
-	from, err := os.OpenFile(mountInfo.DiskPath, unix.O_DIRECT|unix.O_RDONLY, 0)
+	from, err := os.OpenFile(mountInfo.DiskPath, unix.O_DIRECT|unix.O_RDONLY, 0600)
```

Comment:
The mode will be useless here though, no?

Reply:
Yeah, removed that part of the change now so it doesn't look too confusing.